### DISCUSSION TITLE
Is there a working example of storybook with web-components and autodocs / prop table?

### DISCUSSION THREAD

**Discussion by studioab** [storybookjs/storybook#24168] [Documentation]
Everything is working for React out of the box, but I'm following this guide and can't get interface properties to show up in proptable
https://github.com/storybookjs/storybook/blob/next/code/addons/docs/web-components/README.md#props-tables

any working example of such storybook would be very helpful.

thanks.

**Comment by break-stuff** [comment]
Agreed! The docs are really lacking for web components. What framework are you using?

**Comment by mcookdev** [comment]
This is still an issue in v8. Using Lit 3 here.
